### PR TITLE
[CI Visibility] ITR .NET unskippable tests

### PR DESCRIPTION
**Pull request by tonyredondo**
### What does this PR do? What is the motivation?

This PR adds the  documentation for the `unskippable tests` feature in the Intelligent Test Runner for .NET section.

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/continuous_integration/intelligent_test_runner/dotnet.md [hunk 1]
```diff
@@ -54,6 +54,131 @@ In addition to setting the environment variables, you or a user in your organiza
...
+  public void MyTest()
+  {
+    // ...
+  }
+}
+```
+
+#### Test suite
+
+Add a NUnit `PropertyAttribute` with the key `datadog_itr_unskippable` and a non-null value (an string.Empty will work) to your test suite to mark it as unskippable.
```
Comment (romainkomorndatadog) [inline content/en/continuous_integration/intelligent_test_runner/dotnet.md:139]: is it "a `string.Empty`" or "`an string.Empty`" ?

Comment (tonyredondo) [reply content/en/continuous_integration/intelligent_test_runner/dotnet.md]: you know better :) I changed it in [4c4999d](https://github.com/DataDog/documentation/pull/19911/commits/4c4999d9f8607fd39704f95603e378dd4d26054d)

**Review by nikita-tkachenko-datadog — APPROVED** [review]

File: content/en/continuous_integration/intelligent_test_runner/dotnet.md (hunk 1 above)
Comment (romainkomorndatadog) [reply content/en/continuous_integration/intelligent_test_runner/dotnet.md]: With .NET syntax, you never know!

File: content/en/continuous_integration/intelligent_test_runner/dotnet.md [hunk 2]
```diff
@@ -54,6 +54,131 @@ In addition to setting the environment variables, you or a user in your organiza
...
+You can override the Intelligent Test Runner's behavior and prevent specific tests from being skipped. These tests are referred to as unskippable tests.
+
+### Why make tests unskippable?
+
+The Intelligent Test Runner uses code coverage data to determine whether or not tests should be skipped. In some cases, this data may not be sufficient to make this determination.
+
+Examples include:
+
+- Tests that read data from text files
+- Tests that interact with APIs outside of the code being tested (such as remote REST APIs)
```
Comment (drichards-87) [inline content/en/continuous_integration/intelligent_test_runner/dotnet.md:68]:
```suggestion
- Tests that read data from text files.
- Tests that interact with APIs outside of the code being tested (such as remote REST APIs).
```

File: content/en/continuous_integration/intelligent_test_runner/dotnet.md [hunk 3]
```diff
@@ -54,6 +54,131 @@ In addition to setting the environment variables, you or a user in your organiza
...
+  }
+}
+```
+
+{{% /tab %}}
+{{% tab "NUnit" %}}
+
+#### Individual test case
+
+Add a NUnit `PropertyAttribute` with the key `datadog_itr_unskippable` and a non-null value (a string.Empty will work) to your test case to mark it as unskippable.
```
Comment (drichards-87) [inline content/en/continuous_integration/intelligent_test_runner/dotnet.md:121]:
```suggestion
Add a NUnit `PropertyAttribute` with the key `datadog_itr_unskippable` and a non-null value (for example, string.Empty) to your test case to mark it as unskippable.
```

File: content/en/continuous_integration/intelligent_test_runner/dotnet.md [hunk 4]
```diff
@@ -54,6 +54,131 @@ In addition to setting the environment variables, you or a user in your organiza
...
+  public void MyTest()
+  {
+    // ...
+  }
+}
+```
+
+#### Test suite
+
+Add a NUnit `PropertyAttribute` with the key `datadog_itr_unskippable` and a non-null value (a string.Empty will work) to your test suite to mark it as unskippable.
```
Comment (drichards-87) [inline content/en/continuous_integration/intelligent_test_runner/dotnet.md:139]:
```suggestion
Add a NUnit `PropertyAttribute` with the key `datadog_itr_unskippable` and a non-null value (for example, string.Empty) to your test suite to mark it as unskippable.
```

**Review by drichards-87 — APPROVED** [review]
Left you some light feedback from Docs. Let me know when this is ready to be merged. Thanks!

**Pull request merged**